### DISCUSSION TITLE
Hide on scroll for react-tooltip v5?

### DISCUSSION THREAD

**Discussion by moloko** [ReactTooltip/react-tooltip#1030] [Q&A] [answered]
I've just been trying out react-tooltip and really like it - except for one issue I can't seem to solve, which is that the tooltip doesn't immediately disappear if you scroll.

I think it's likely a similar issue to #746 given that it seems to work fine on the examples in the docs. But I can't use the same `isCapture` solution that was given for #746 as it doesn't seem to be available in v5.

I notice that [here](https://react-tooltip.com/docs/upgrade-guide/changelog-v4-v5#v4-props-available-in-v5) it says 'scrollHide - can be implemented if requested' - so maybe it's that which is needed?

**Comment by gabrieljablonski** [comment, marked as answer]
I've opened a feature request for this at #1031.

It might be a few days before we can get to this, so if you'd like a hacky workaround until then, I could give some suggestions.

**Reply by moloko** [reply]
That's great, thanks for the quick response! And thanks for the offer but no problem waiting for proper solution.
